cdhweb/people/templates/people/profile_list.html

```diff
@@ -7,7 +7,7 @@
 {% block main %}
 <section class="people">
 
-<h1>{{ title }}</h1>
+<h1>{{ current_title }}</h1>
```

Comment:
There will be times of the year that there are no upcoming speakers, so we need some language to display when this section is empty. Maybe we can borrow/adapt from the event list? Could link to main event page (which includes recent past events).
> Next semester's events are being scheduled.  Check back later or view past events.

I think this might be specific to speakers - it seems less likely that the other categories will ever have no current members (maybe possible for faculty affiliates and students?).

Reply:
Adapted the event page language and styled it accordingly.